web/spec/connect-web.callback.spec.ts

```diff
@@ -330,10 +332,10 @@ describe("connect_web_callback_client", function () {
     });
     client.failUnaryCall({}, (err: ConnectError | undefined) => {
       expect(err).toBeInstanceOf(ConnectError);
-      expect(err?.code).toEqual(StatusCode.ResourceExhausted);
+      expect(err?.code).toEqual(Code.ResourceExhausted);
       expect(err?.rawMessage).toEqual("soirée 🎉");
       expect(err?.details.length).toEqual(1);
-      expect(err?.details[0].equals(Any.pack(expectedErrorDetail))).toBeTrue();
+      expect(err?.details[0].equals(expectedErrorDetail)).toBeTrue();
```

Comment:
I think this would be slightly more robust if we called the equals method on the known good value:

```suggestion
      expect(expectedErrorDetail.equals(err?.details[0])).toBeTrue();
```